README.md

```diff
@@ -145,11 +145,13 @@ type Message = StartMessage | DefineMessage | EvaluateMessage | AnalysisMessage;
 
 interface DefineResponse extends Base {
   success: boolean;
+  error?: string;
```

Comment:
Can you name this `message` instead, for consistency with `"analysis"` messages?

```suggestion
  message?: string;
```

Or, alternatively, if you'd prefer to name this `error`, could you also rename the `message` field in `"analysis"` messages from `"message"` to `"error"`?

Reply:
I will rename them all to `"error"`; on average this is more evocative of what is actually going on.